src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -166,7 +166,8 @@ function warnAndMonitorForKeyUse(message, element, parentType) {
 
   warning(
     false,
-    message + '%s%s See https://fb.me/react-warning-keys for more information.',
+    '%s%s%s See https://fb.me/react-warning-keys for more information.',
```

Comment:
The two callers of this function have different warning configs internally (static_upstream/core/createWarning.js) so we can't sync it like this without changing behavior. We should just split this out into two separate warning calls probably – this code is a little overabstracted.